<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>314</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Фоносемантический анализатор </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../University/$Магистратура/Program/Ashampoo Snap 8/Mouse/OnClick/mouse-swirl_white.png</normaloff>../../University/$Магистратура/Program/Ashampoo Snap 8/Mouse/OnClick/mouse-swirl_white.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout1">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>15</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="btn_NormalizeFace">
              <property name="text">
               <string>Нормальное положение лица</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QPushButton" name="btn_leftCloseEye">
                <property name="text">
                 <string>Левый глаз закрыт</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_rightCloseEye">
                <property name="text">
                 <string>Правый глаз закрыт</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_browUp">
                <property name="text">
                 <string>Подняты Брови</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="btn_lipsOpen">
                  <property name="text">
                   <string>Рот открыт</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="btn_smile">
                  <property name="text">
                   <string>Улыбка</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QPushButton" name="btn_liftHead">
                <property name="text">
                 <string>Подъем головы</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_turnHead">
                <property name="text">
                 <string>Поворот головы</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btn_tiltHead">
                <property name="text">
                 <string>Наклон головы</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btn_FaceReconstruction">
              <property name="text">
               <string>Реконструкция лица</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_SwichSendUDP">
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Включить передачу данных</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_SwichSendVideo">
              <property name="text">
               <string>Включить камеру</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="frameVideo">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder></zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
